docs/pages/push-notifications/push-notifications-setup.mdx

```diff
@@ -90,68 +93,92 @@ async function registerForPushNotificationsAsync() {
       lightColor: '#FF231F7C',
     });
   }
-  /* @end */
 
-  /* @info You should make sure the app is running on a physical device since push notifications don't work on an emulator/simulator. */
   if (Device.isDevice) {
-    /* @end */
-    const { status: existingStatus } = await Notifications.getPermissionsAsync();
+    const { status: existingStatus } =
+      await Notifications.getPermissionsAsync();
     let finalStatus = existingStatus;
     if (existingStatus !== 'granted') {
       const { status } = await Notifications.requestPermissionsAsync();
       finalStatus = status;
     }
     if (finalStatus !== 'granted') {
-      alert('Failed to get push token for push notification!');
+      handleRegistrationError('Permission not granted to get push token for push notification!');
       return;
     }
-    /* @info This provides the ExpoPushToken which is attributed based on the ID of the project. */
-    token = await Notifications.getExpoPushTokenAsync({
-      projectId: Constants.expoConfig.extra.eas.projectId,
-    });
+    const projectId =
+      Constants?.expoConfig?.extra?.eas?.projectId ??
+      Constants?.easConfig?.projectId;
+    if (!projectId) {
+      handleRegistrationError('Project ID not found');
+    }
+    try {
+    /* @info This provides the ExpoPushToken using the unique project ID. */
```

Comment:
IMO this could be clearer by saying something like `This fetches the ExpoPushToken (if not previously fetched), which is unique to this device and projectID`.

Reply:
```suggestion
    /* @info This fetches the ExpoPushToken (if not previously fetched), which is unique to this device and projectID. */
```